src/views/RecIM/components/Listing/index.js

```diff
@@ -0,0 +1,47 @@
+import { Grid } from '@material-ui/core/';
+import styles from './Listing.module.css';
+import { Link } from 'react-router-dom';
+
+const LeagueListing = ({ leagueID }) => {
+  return (
+    <>
+      <Link to={`/recim/league/${leagueID}`} className="gc360_link">
+        <Grid container className={styles.listing}>
+          <Grid item>League Listing</Grid>
+          {/* include: 
+          - activity type (league, tournament, one-off)
+          - registration deadline IF there is one (start date as well for admin only)
+          - date(s) of activity (ex. season date range or tournament date)
+          */}
+        </Grid>
+      </Link>
+    </>
+  );
+};
+
+const TeamListing = ({ leagueID, teamID }) => {
+  return (
+    <>
+      <Link to={`/recim/league/${leagueID}/team/${teamID}`} className="gc360_link">
+        <Grid container className={styles.listing}>
+          <Grid item>Team Listing</Grid>
+        </Grid>
+      </Link>
+    </>
+  );
+};
+
+// We could also use ParticipantID (not student ID) if we have that and prefer it to AD_Username
+const ParticipantListing = ({ username }) => {
+  return (
+    <>
+      <Link to={`/recim/profile/${username}`} className="gc360_link">
```

Comment:
Question for your team: would it make sense to incorporate this RecIM user info into the regular user profile? I'm imagining a new card on the profile page that has RecIM info attached to, similar perhaps to the current Involvements list?

I don't know your requirements or what information is included in the RecIM user's profile. I do, though, think it would be nice to have one unified profile rather than dispersing a user's info across the site.

Reply:
I'm not sure exactly why we have a participant profile page? I thought that we weren't going to have profiles for each person, one of the reasons against it was that we would most likely need to implement some sort of search feature for RecIM participants which would conflict/look weird with people search already existing in the 360 Header. 

Unless this is a typo and you meant to send directly to the 360 user profile (line 38). Though in that case, I would argue that it might be a hassle to need to get back to where you saw the people listing. 
i.e.
Press RecIM -> Activity -> Team -> Person

To answer @EjPlatzer's question, I don't think that is too hard to do. But it might be excess information that would further clutter the Profile view as it is. Moreover, we would have to handle cases where PERSON has not signed up for RecIM in which we might just have an empty card which might look awkward. I'm also pretty sure that our customer is not putting individual statistics high on their priority list, so the card would be relatively empty (Though this might be subject to change.)

Otherwise, @cpabbot code looks good.